app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registeratable, :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] You have a type here should be `registeratable` => `registerable` kindly fix it